netbox/dcim/tables/devices.py

```diff
@@ -557,6 +557,12 @@ class BaseInterfaceTable(NetBoxTable):
         orderable=False,
         verbose_name=_('IP Addresses')
     )
+    virtual_ips = tables.TemplateColumn(
+        accessor=Accessor('fhrp_group_assignments'),
```

Comment:
```suggestion
        accessor=Accessor('fhrp_group_assignments__ip_addresses'),
```

We should reference the IP assignments directly to ensure they are prefetched when the table column is displayed.